### PR TITLE
Feat(UI) #9204: Added preview panel for all the data assets.

### PR DESCRIPTION
**Pull request by aniketkatkar97**
### Describe your changes :

I worked on #9204 

- [x] Adding data assets summary details panel for all the data assets.
- [x] Styling improvements
- [x] bug fixes

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">

1. Tables:
Screenshot 2022-12-16 at 11 51 30 AM

2. Dashboards:
Screenshot 2022-12-16 at 11 51 39 AM

3. Pipelines:
Screenshot 2022-12-16 at 11 51 45 AM

4. MlModels:
Screenshot 2022-12-16 at 11 51 54 AM

</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

**Comment by chirag-madlani** [comment]
@aniketkatkar97 Did we also make the first result selected by default?

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/EntitySummaryPanel.component.tsx [hunk 1]
```diff
@@ -12,109 +12,59 @@
...
+      return <TableSummary entityDetails={entityDetails.details as Table} />;
+    } else if (entityDetails.entityType === ExplorePageTabs.TOPICS) {
+      return <TopicSummary entityDetails={entityDetails.details as Topic} />;
+    } else if (entityDetails.entityType === ExplorePageTabs.DASHBOARDS) {
+      return (
+        <DashboardSummary entityDetails={entityDetails.details as Dashboard} />
+      );
+    } else if (entityDetails.entityType === ExplorePageTabs.PIPELINES) {
+      return (
+        <PipelineSummary entityDetails={entityDetails.details as Pipeline} />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/EntitySummaryPanel.component.tsx:50]: use switch case for such case

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SummaryList/SummaryList.interface.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { ReactNode } from 'react';
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/SummaryList/SummaryList.interface.ts:1]: Should be after licence

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/TableSummary/TableSummary.component.tsx [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+          entityName: entityDetails.name,
+        })
+      );
+    }
+  };
+
+  const overallSummary: OverallTableSummeryType[] = useMemo(() => {
+    return [
+      {
+        title: 'Row Count',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/TableSummary/TableSummary.component.tsx:116]: Missed localisation here?

**Review by chirag-madlani — APPROVED** [review]

**Review by chirag-madlani — DISMISSED** [review]

**Comment by aniketkatkar97** [comment]
> @aniketkatkar97 Did we also make the first result selected by default?
Yes we did, also the selected card will remain highlighted

**Comment by aniketkatkar97** [comment]
> @devyani-kaushik the side bar headings compare to the table name heading is bigger. May be we should make it smaller to get attention to the table name?

@harshach, Changed the styling for the summary title also, updated the screenshots. Can you please check?

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**